src/ui/components/EventsTimeline.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import ReactDOM from "react-dom";
+
+import "./EventsTimeline.css";
+
+export default function EventsTimeline() {
+  return (
+    <div className="events-timeline">
+      <div>
+        <div className="event ">
+          <div className="img rxjs"></div>
+          <div className="label">I recorded a quick todo list with 3 completed todos</div>
```

Comment:
How's this meant to get populated or be dynamic?

Reply:
:) yeah, we have to pull the data from somewhere